Settings/Help: links are clickable. Fixes #232.

rgmf opened the pull request:
**Describe the pull request**
Links already clickable. I've tested it in API 21 and API 29.

With android:autoLink="web" in xml layout only links like this works:
`https://github.com/OpenTracksApp/OpenTracks`

but not links like this:
`<a href="https://github.com/OpenTracksApp/OpenTracks">OpenTracks</a>`

So I've decided to use `setMovementMethod` method on `TextView` with links.

**Link to the the issue**
#232 

**License agreement**
By opening this pull request, you are providing your contribution under the _Apache License 2.0_ (see [LICENSE.md](LICENSE.md)).

dennisguse commented:
Harding the text view is problematic from a maintenance perspective as we might add more (containing links) while forgetting to make them clickable.

What about a recursive approach?

Completely untested:
```
package de.dennisguse.opentracks;

import android.os.Bundle;
import android.text.method.LinkMovementMethod;
import android.view.View;
import android.view.ViewGroup;
import android.widget.LinearLayout;
import android.widget.TextView;

import androidx.annotation.Nullable;

public class HelpActivity extends AbstractActivity {

    @Override
    public void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        LinearLayout view = findViewById(getLayoutResId());
        makeClickableLinks(view);
    }

    @Override
    protected int getLayoutResId() {
        return R.layout.help;
    }


    public static void makeClickableLinks(View view) {
        if (view instanceof TextView) {
            ((TextView)view).setMovementMethod(LinkMovementMethod.getInstance());
            return;
        }
        if (view instanceof ViewGroup) {
            makeClickableLinks(view);
        }
    }
}
```

rgmf commented:
Changes:
- Set movement method for every TextView (with or without links). Traverse from root view to find all TextView.
- Fixed the same issue in AboutActivity.
- You couldn't see About on small screens. Added Scroll. Fix this hidden bug.

Tested in API 21 and 25.

dennisguse commented:
Extracted method to a utils class.

@rgmf Squash and merge :)

dennisguse commented:
:smiley:
